File: documentation/user/en/query/filtering/examples/range/attribute-in-range.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeInRange(
						"validity", 
						OffsetDateTime.parse("2023-12-05T12:00:00+01:00", DateTimeFormatter.ISO_OFFSET_DATE_TIME)
					)
				),
				require(
					entityFetch(
						attributeContent("code")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/range/attribute-in-range.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeInRange(
        			"validity", 
        			DateTimeOffset.Parse("2023-12-05T12:00:00+01:00", DateTimeFormatInfo.InvariantInfo)
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code")
        		)
        	)
        )
	)
);